xbmc/Application.h

```diff
@@ -297,6 +297,11 @@ class CApplication : public CXBApplicationEx, public IPlayerCallback, public IMs
 #endif
 
   CApplicationPlayer* m_pPlayer;
+  /*!
+   \brief Returns the application player
+   \return The application player
+   */
+  CApplicationPlayer* GetPlayer() { return m_pPlayer; }
```

Comment:
m_pPlayer is not supposed to be exposed. why do you access the internals? applicationplayer has a method SeekTimeRelative